assignments/closure.js

```diff
@@ -1,21 +1,52 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+var name = "Dora";
+function sayHello() {
+  return "Hey, my name is " + name + "!";
+}
```

Comment:
This is not actually a closure, because we are just using a variable in the global scope. The idea of a closure is that scopes continue to be available to functions defined inside of them, even if that scope seems to be otherwise discarded. For example,

function coolest(name) {
  return function() {
    console.log('The coolest is' + name);
  }
}
const dak = coolest('dak');

When we call dak() it will still use the name value property because it is stored within that functions closure, even though that scope is no longer available.